ipwb/serviceWorker.js

```diff
@@ -1,76 +1,39 @@
 /* eslint-env serviceworker */
 
-let baseDatetime = ''
-
-function getMyVersion () {
-  fetch(self.location.href)
-  .then(function (resp) {
-    console.log('Running ServiceWorker version ' + resp.headers.get('Server'))
-  })
-}
-
+// This makes a module available named "reconstructive"
+importScripts('https://oduwsdl.github.io/reconstructive/reconstructive.js')
```

Comment:
This causes the SW to fail in offline testing. Please use the reconstructive local to the replay system.

Reply:
This has been taken care of. Please test again to see if things are behaving expected.